.circleci/config.yml

```diff
@@ -48,13 +59,24 @@ jobs:
     working_directory: /go/src/github.com/grafana/grafana
     steps:
         - checkout
+        - run:
+            name: ci job started
+            command: './scripts/ci-job-started.sh'
```

Comment:
Maybe we should only wrap the part of the jobs we have control over? `apt update`, ` restore-cache`, `yarn install` etc might cause the build to fail but this is sth we don't have control over. Sort of depends on what we want to see from these metrics.

Reply:
I wanted to keep it simple so didn't think to much about this.